specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-10-01-preview/workspaceRP.json

```diff
@@ -5347,6 +5492,31 @@
         }
       }
     },
+    "NetworkAcls": {
+      "type": "object",
+      "properties": {
+        "bypass": {
+          "type": "string"
+        },
+        "defaultAction": {
+          "description": "The default action when no rule from ipRules and from virtualNetworkRules match. This is only used after the bypass property has been evaluated.",
+          "$ref": "#/definitions/DefaultActionType"
+        },
+        "ipRules": {
+          "description": "Rules governing the accessibility of a resource from a specific ip address or ip range.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IPRule"
+          }
+        },
+        "resourceAccessRules": {
```

Comment:
>resourceAccessRules [](http://example.com/codeflow?start=9&length=19)

Add a description also for this property

Reply:
removed as this is not ready